### DISCUSSION TITLE
Address, Port or Protocols Bytes/Packets Aggregation

### DISCUSSION THREAD

**Discussion by islameehassan** [GyulyVGC/sniffnet#547] [Ideas]
The latest release allows for filtering over addresses, ports or protocols, so you can inspect the filtered connections. However, I believe a more general and holistic overview of a given address, port or protocol should be available. Namely, the user should be able to get the total packets and bytes sent to and from an address, port or protocol - a row in the table represents an address and its total bytes/packets in and out. You can switch between this general view and the original detailed view in the inspect page. This would even ease the implementation of bandwidth throttling - which is another idea that, i think, should be considered. In this case, the user can see the total bandwidth in one row for a given address, and just click on this address to throttle.

**Comment by GyulyVGC** [comment]
Thanks for the idea.
This is something that requires some more careful investigations before starting with a possible implementation.
If I understood right, you'd like to see something like the data in the lower portion of the _Overview_ page but grouped by more parameters (e.g., port, address, protocol) other than network host and service.
Feel free to open a more detailed issue also including sketches of what you had in mind when you find time.
However, take your time since this is a feature that would require another major release and thus isn't something that will land soon.

**Comment by GyulyVGC** [comment]
Closing since this was implemented in more recent Sniffnet versions.
